nixos/modules/services/networking/syncthing.nix

```diff
@@ -7,6 +7,21 @@ let
   cfg = config.services.syncthing;
   defaultUser = "syncthing";
```

Comment:
It's customary for those user- and group names to have an entry in `nixos/modules/misc/ids.nix`.